fe/fe-core/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -117,7 +117,7 @@ public static class Properties {
     // ---------------------------- for old insert stmt ----------------------------
 
     public boolean isValuesOrConstantSelect() {
-        throw new UnsupportedOperationException("only invoked in NativeInsertStmt");
+        return false;
```

Comment:
why change this?

Reply:
beacuse of the insert stmt as the abstract class, execute to this point[1] will throw exception when i tested. But I think this isn't correct.

[1] https://github.com/apache/doris/blob/master/fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java#L324